prepareCourseForLadok/index.js

```diff
@@ -226,11 +228,7 @@ async function start () {
     const assignment = assignments.find(a => a.integration_data.sis_assignment_id === assignmentSisID)
     const assignmentName = `LADOK - ${examinationRound.examCode}`
 
-    const { modulId } = await inquirer.prompt({
-      name: 'modulId',
-      type: 'input',
-      message: `Enter the ladok id for the module '${examinationRound.examCode} ${examinationRound.title}'`,
-      default: assignment && assignment.integration_id })
+    const modulId = examinationRound.ladokUID
```

Comment:
This constant is seemingly never used. Either remove it or refer to it on line 241.

Reply:
Good point! I removed that, and all the other unused functions

Reply:
Goodie!